opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/RuntimeEnvironment.java

```diff
@@ -125,14 +126,16 @@
     private RuntimeEnvironment() {
         configuration = new Configuration();
         configLock = new ReentrantReadWriteLock();
+        authFrameworkLock = new ReentrantLock();
```

Comment:
i think with this you can just use `new Object()` and synchronize on that object